inc/targetbase.class.php

```diff
@@ -1048,11 +1048,123 @@ protected function parseTags($content, PluginFormcreatorFormAnswer $formanswer,
                }
             }
          }
+
+         if ($question->getField('fieldtype') == 'glpiselect') {
+            $content = $this->parseObjectProperties(
+               $question,
+               $questionId,
```

Comment:
This is the ID of the questoin given in 1st argument. I think this is useless to pass it here.